Source/Widgets/Viewer/Viewer.js

```diff
@@ -141,6 +142,15 @@ function trackDataSourceClock(timeline, clock, dataSource) {
     if (defined(dataSourceClock)) {
       dataSourceClock.getValue(clock);
       if (defined(timeline)) {
+        if (
+          JulianDate.equals(dataSourceClock.startTime, dataSourceClock.stopTime)
+        ) {
+          JulianDate.addSeconds(
+            dataSourceClock.startTime,
+            5,
+            dataSourceClock.stopTime
+          );
+        }
```

Comment:
Add a comment in the code explaining what the fix is doing and why it's needed. This will help future readers understand what's going on.